setup.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (C) 2020 Dominik Tuchyna
```

Comment:
This file should not be needed. This is not a package released on PyPI.

Reply:
what about the pre-commit `This is not a Python project (no setup.py).` ?

Reply:
@fridex  and shouldn't be the setup.py be also present when doing s2i ?

Reply:
you can remove the setup.py check from pre-commit configuration, setup.py will not be used in s2i

Reply:
you can remove the setup.py check from pre-commit. setup.py will not be used in s2i.